src/utils.ts

```diff
@@ -7,10 +7,10 @@ export function useUnmount(fn: () => void) {
 }
 
 export function useForceUpdate() {
-    const [, setTick] = useState(null)
+    const [, setTick] = useState(0)
 
     const update = useCallback(() => {
-        setTick(null)
+        setTick(tick => tick + 1)
```

Comment:
I know this is most likely not an issue at all, but we "only" have up to 9007199254740991 (Number.MAX_SAFE_INTEGER) renders until this breaks :)
Might be worth to set 0 once that value is reached?

Reply:
Um, would really like to see an app that would have a single component rendered that long to waste that :D I am mostly hoping that some official way of `forceUpdate` will rise eventually.

Reply:
Makes me wonder if flipping a boolean would be enough or if react is smart enough so if the bacthed state is not changed it won't re-render

Reply:
Either way, given a component is rendering 60 times per second it will take 285616414 years more or less for it to break, so it was just a stupid comment :)